tests/zstreamtest.c

```diff
@@ -210,9 +210,9 @@ static int basicUnitTests(U32 seed, double compressibility, ZSTD_customMem custo
     /* context size functions */
```

Comment:
Do we have a tests that make sure the prefix is only used once for block/streaming compression?